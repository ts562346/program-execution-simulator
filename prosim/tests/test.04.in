test4 3
DOOP 3
BLOCK 5
HALT

