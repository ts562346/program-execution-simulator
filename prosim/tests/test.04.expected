00000: DOOP
00003: BLOCK
00008: HALT

DOOP count : 1
DOOP time  : 3
BLOCK count: 1
BLOCK time : 5
